server/authz/policy.rego

```diff
@@ -595,37 +595,37 @@ allow {
 }
 
 ##
-# Apple MDM
+# Apple and Windows MDM
 ##
 
-# Global admins and maintainers can read and write Apple MDM config profiles.
+# Global admins and maintainers can read and write MDM config profiles.
 allow {
-  object.type == "mdm_apple_config_profile"
+  object.type == "mdm_config_profile"
```

Comment:
Creating this PR immediately so we can all have the new platform-independent authorization for MDM profiles. It will likely be required to unblock other tickets in this story.